assignments/2. variables/index.md

```diff
@@ -41,12 +46,13 @@ Solve this using mathematical operations. (+, -, *, / , etc)
 ```js
 var amount = 2080;
 // Define a new variable and store the value that is 80 less then the value of amount.
-
+let subtractedAmount = (2080 - 80);
 // Define a new variable and store the value that is 200 more then the value of amount.
-
+let addedAmount = (2080 + 200);
```

Comment:
use the variable name i.e `amount` instead of value 280 to calculate the other values.

Reply:
that was a blunder